### ISSUE TITLE
Invalid configuration: Unexpected item 'parameters › disallowedFunctionCalls'.

### ISSUE THREAD

**Issue by k00ni** [spaze/phpstan-disallowed-calls#148]
I am calling PHPStan (v**1.9.2**, using **PHP 8.1.6**) with:
> vendor/bin/phpstan analyse -c phpstan.neon

and I get the following error when using one of your neon-files:
> Invalid configuration:
> Unexpected item 'parameters › disallowedFunctionCalls'.

My **PHPStan config** is:

```yaml
includes:
    - vendor/phpstan/phpstan-phpunit/extension.neon
    - vendor/phpstan/phpstan-strict-rules/rules.neon
    - vendor/spaze/phpstan-disallowed-calls/disallowed-dangerous-calls.neon
    # - vendor/phpstan/phpstan/conf/bleedingEdge.neon

parameters:
    bootstrapFiles:
        - vendor/autoload.php
    scanFiles:
      - config/bootstrap.php

    fileExtensions:
        - php

    level: 8

    paths:
        - src
        - test

    parallel:
        maximumNumberOfProcesses: 2

    tmpDir: ./var/cache/phpstan

    reportUnmatchedIgnoredErrors: true

    checkGenericClassInNonGenericObjectType: true

    ignoreErrors:
        - '/Variable method call on/'
        - '/Variable property access on DateInterval/'
        - '/Variable property access on App\\Service\\Statistics\\Class\\OrderPack/'
```

When only using `vendor/spaze/phpstan-disallowed-calls/extension.neon` it doesn't report anything (tested it with a blunt `var_dump`).

I would like to check out your PHPStan rules, but can't get it to work. Help is appreciated.

**Comment by spaze** [comment]
You need to load both `extension.neon` and say `disallowed-dangerous-calls.neon`, see my [example](https://github.com/spaze/michalspacek.cz/blob/a0a043821197a6f2243abacc873809be577fcb13/site/disallowed-calls.neon#L21-L26).

Loading just `extension.neon` doesn't report anything because you may want to have only your own disallowed config, and loading just `disallowed-dangerous-calls.neon` doesn't load the extension config because you may want to include multiple `disallowed-*-calls.neon` files.

Let me know if that helps and if yes, please close the issue :-)

**Comment by k00ni** [comment]
Thanks, that works!
